packages/vitest/src/utils/retry.ts

```diff
@@ -0,0 +1,70 @@
+import type { SerializedConfig } from '../runtime/config'
```

Comment:
This whole file should be in `packages/runner`. There is no reason to keep it in another package